.github/workflows/actions.yml

```diff
@@ -49,3 +49,13 @@ jobs:
               --release true \
               --signed true
           fi
+      - name: Create GitHub Release
+        id: create_gh_release
+        uses: actions/create-release@latest
```

Comment:
Is it a good idea to use `@latest` ?
What if they do a breaking release and change the API?
Isn't it better to stick to a tag (even major could work)?

Reply:
Makes sense. I replaced the version with a concrete tag.

Reply:
I'll also update other PRs once this one is merged.